src/components/forms/transaction/txFormActions.ts

```diff
@@ -0,0 +1,11 @@
+import { Split } from '@/book/entities';
+import { TransactionFormContext } from './txFormOps';
+
+const onAddSplit = ({ splitsFieldArray }: TransactionFormContext) => {
+  const newSplit = Split.create({ value: 0, quantity: 0 });
+  return splitsFieldArray.append(newSplit);
+};
```

Comment:
nit: I've been coding the whole project to use the `export function onAddSplit ...` can we keep it consistent? (if there is a pro/con over one or the other I'm happy to talk about it!)

Reply:
Sure, I can update that.

The idea is to have a namespace that communicates the domain of the function. Eg. `addOnSplit(foo)` could do anything, however `txFormActions.onAddSplit(foo)` is clear that it works with the transaction form.
It's possible to `import * as txFormActions from '...'` however that's optional and therefore not necessarily consistent, esp when auto import steps in to help

Reply:
to me, where it is defined and how it is imported is enough, otherwise you find duplicating naming and stuff a lot for stuff that is not even intended to be reused outside its local scope. `forms/transaction` can be considered a local package and its functions local so IMO something like `forms/transaction/actions.ts` is something that contains actions for this package